extensions/pubmed/type-check/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "type-check",
+  "version": "0.4.0",
```

Comment:
This appears to be the published type-check library available here: npmjs.com/package/type-check – is it possible to depend on this instead of vendoring the package here?

Reply:
I have no idea how this folder got there in the first place. Deleted it now (again with the egg dance of deleting via the github website, then `npx ray pull-contributions`, then adding a change again so there is something new and `push` and `publish` goes through - is there any way to make this easier?): [https://github.com/raycast/extensions/pull/5843](https://github.com/raycast/extensions/pull/5843)

Reply:
@justspacedog nice one. I'm afraid I'm not familiar with the workflow built in to `ray`, but it looks like all it's actually doing is copying around files between the files you're editing, and a fork of the Raycast extensions codebase. You can just clone this repo and treat it like any other git repo in terms of forks/pull requests/etc. You may find that easier.

By the way, in case it wasn't clear, I only left this comment as a drive-by as I was working on some extension fixes myself today and thought this sounded interesting. I'm not actually able to review this for merging, but I'm sure someone from the Raycast team will take a look soon!

Reply:
Thanks for the tip anyway!

I'm not a full time developer (but a medical student) and I'm having trouble working on the raycast/extensions repository. When I delete files on my site, the files in my repository are deleted, but the deletions (of files) are somehow not transferred to raycast/extensions.

Reply:
> I'm not a full time developer (but a medical student)

Could have fooled me! Good going with this extension, it's great to see!

> When I delete files on my site, the files in my repository are deleted, but the deletions (of files) are somehow not transferred to raycast/extensions

Are you writing on GitHub, or do you have a local clone of the repo on your computer? In the same way that you commit a file change, you also need to commit a file _deletion_, they're treated the same as changes. You should be able to push commits to the same branch you're already using (which might be your `main` branch).

Reply:
> Could have fooled me! Good going with this extension, it's great to see!

Hahaha thanks, but it's just scripting - there are already many extensions with source code and many modules you can just use.

> Are you writing on GitHub, or do you have a local clone of the repo on your computer? In the same way that you commit a file change, you also need to commit a file _deletion_, they're treated the same as changes. You should be able to push commits to the same branch you're already using (which might be your `main` branch).

Ah that was my mistake! I only ever typed `git add .` (never `git rm`) into the command line. :D